### PR TITLE
feat(Command Managers): Implement Command Handling

### PR DESCRIPTION
**Pull request by PenPow**
Managers to handle command handling

### CODE REVIEW

**Comment by KyomiDev** [comment]
This requires a rebase.

File: src/events/ready.js [hunk 1]
```diff
@@ -20,24 +18,21 @@ class Ready {
...
+		// 	commandArray.push(command.generateSlashCommand());
 
-			table.addRow(command.name, this.client.utils.capitalize(command.type), 'Pass');
-		});
+		// 	table.addRow(command.name, this.client.utils.capitalize(command.type), 'Pass');
+		// });
 
 		// const test_guild = await this.client.guilds.fetch('789215359878168586');
 		// test_guild.commands.set(commandArray);
-		this.client.logger.log(table.toString());
```
Comment (KyomiDev) [inline src/events/ready.js:36]: Remove this from the event and create a dedicated deploy command in a new PR.

File: src/events/ready.js [hunk 2]
```diff
@@ -20,24 +18,21 @@ class Ready {
...
 
 		this.client.user.setPresence({
```
Comment (KyomiDev) [inline src/events/ready.js:37]: Can we replace this with a fixed status as per discord developer TOS

**Review by KyomiDev — CHANGES_REQUESTED** [review]
Small Nits, other than that LGTM!

File: src/events/ready.js (hunk 2 above)
Comment (PenPow) [reply src/events/ready.js:37]: 👍

File: src/events/ready.js (hunk 1 above)
Comment (PenPow) [reply src/events/ready.js:36]: 👍

File: src/events/ready.js (hunk 2 above)
Comment (KyomiDev) [reply src/events/ready.js:37]: Oops, selected wrong line, see the lines below

File: src/events/ready.js (hunk 1 above)
Comment (PenPow) [reply src/events/ready.js:36]: Added in PR #6

**Review by KyomiDev — APPROVED** [review]

**Pull request merged**